aboutPage.html

```diff
@@ -0,0 +1,82 @@
+</head>
+<body>
+    <header>
+        <nav>
+            <ul class="topNav">
+                <li><img class="icon-facebook" src="./images/facebook-f (2).svg" alt=""></li>
+                <li><img class="icon-twitter" src="./images/twitter (1).svg" alt=""></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+            <div class="navBar">
+                <span class="logo"><a href="./index.html"><img class="logoImg" src="./images/Global-Summit2022.png" alt=""></a></span>
+                <ul class="bottomNav">
+                    <li><a class="menuItem" href="./index.html">Home</a></li>
+                    <li><a class="menuItem" href="#">Program</a></li>
+                    <li><a class="menuItem" href="#">Join</a></li>
+                    <li><a class="menuItem" href="#">Sponsor</a></li>
+                    <li><a class="menuItem" href="#">News</a></li>
+                    <li><button type="submit" class="menuItem" id="btn">IAM Campaign</button></li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="homePage">
+            <button type="submit" class="menuIcon"><img src="images/bars-solid.svg" alt="Menu" /></button>
+            <button type="submit" class="closeIcon"><img src="images/close.png" alt="CloseIcon" /></button>
+            <div class="mainTitle">
+                <h3 class="hello">"Hello! Mental Athletes"</h3>
+                <h1>International Association of Memory Global Summit 2022</h1>
+            </div>
+            <p class="description">The IAM Global Summit brings together the community of memory experts, mental athletes, and activists who are willing to enhance their memory skills and understand how their brain work. This year, we hope to expand our invitation list including organizations and individuals.</p>
+            <p class="contact">Please contact us per Email for any furthur questions about IAM Global Summit 2022!</p>
+            <p class="email">iamsummit2022@iam.org</p>
+        </section>
+        <section class="summitLogo">
+            <h3 class="h3_logo">International Association of Memory Global Summit 2022 Logo</h3>
+            <hr class="hr_logo">
+            <p class="selectionLogo">The logo of IAM Global Summit 2022 was decided through the logo competition from 8. October to 7. November.</p>
+            <div class="logoImage"><img id="logoImg" src="./images/Global-Summit2022.png" alt=""></div>
+        </section> 
+        <hr class="divider">
+        <section class="summitPast">
+            <h3 class="h3_logo">See the past championships</h3>
+            <hr class="hr_logo">
+            <p class="selectionLogo">Take a look at the last two Memory championships which took place in Indonesia 2017 and China 2019</p>
+            <div class="imgChamp">
+                <div class="firstChamp"></div>
+                <div class="secondChamp"></div>
```

Comment:
- [ ] Your pictures should have text explaining the events that have happened and should be centered as it is designed from Behance design. Apply to both mobile and desktop screens.

Your design 
![image](https://user-images.githubusercontent.com/43877689/197261572-dcf0c7d4-571d-40a1-8ff0-25f1c6916267.png)

Behance design 
![image](https://user-images.githubusercontent.com/43877689/197261680-2fac43b3-9af5-4ec8-a087-8240e952c4a1.png)